### PR TITLE
Redshift sequences

### PR DESCRIPTION
**Pull request by jseabold**
## Todos
- [x] MIT compatible
- [x] Tests
- [x] Documentation
- [x] Updated CHANGES.rst

@graingert See if this addresses your comments in #85.

> make a new PR that sets the flags and uses my test

Sorry, I'm not sure what you mean by 'sets the flags.'

### CODE REVIEW

File: sqlalchemy_redshift/dialect.py [hunk 1]
```diff
@@ -317,6 +319,9 @@ class RedshiftDialect(PGDialect_psycopg2):
     """
 
     name = 'redshift'
+    supports_sequences = False
```
Comment (graingert) [inline sqlalchemy_redshift/dialect.py]: these are "the flags"

**Comment by graingert** [comment]
you'll have to squash this and you can't include any commits by @mtambos as they have not ticked the MIT box.

File: sqlalchemy_redshift/dialect.py (hunk 1 above)
Comment (jseabold) [inline sqlalchemy_redshift/dialect.py]: I see. What do you want done here?

Comment (graingert) [inline sqlalchemy_redshift/dialect.py]: I need you to include these flags so that the new test you added passes.

Comment (jseabold) [inline sqlalchemy_redshift/dialect.py]: Sorry you're going to have to walk me through this. Include these flags where? It's not obvious to me glancing at the test fixtures.

Comment (graingert) [inline sqlalchemy_redshift/dialect.py]: these are the flags, include them here. Like you have

**Comment by graingert** [comment]
@jseabold Looks like that didn't work: https://travis-ci.org/sqlalchemy-redshift/sqlalchemy-redshift
The id is 0 each time.

File: sqlalchemy_redshift/dialect.py (hunk 1 above)
Comment (jseabold) [inline sqlalchemy_redshift/dialect.py]: Ok, I saw them still skipped in travis so I wasn't sure.

**Comment by jseabold** [comment]
Hmm, ok. I'll have to use my brain. Must be a test issue, because this works for me.

**Comment by graingert** [comment]
They're skipped for PRs because the tests have to boot a real redshift
instance

**Comment by jseabold** [comment]
I just tried this on my cluster and it works. Redshift auto increments according to the number of slices on the cluster I believe, so you're not guaranteed to 0, 1, 2, 3, 4 here, depending on what your cluster looks like.

I think this is just a broken test and the expectations are wrong here. My sqlalchemy-fu isn't strong enough to explain why but I need to commit the session rather than flushing it and you should test the number of distinct values returned. So something like

```
session.commit()
assert len(selected_distinct_ids) == 5
```

**Comment by graingert** [comment]
you won't need a commit, flush should work.

**Comment by graingert** [comment]
```python
def test_postfetch_lastrowid(redshift_session):
    session = redshift_session
    postfetch_examples = [models.PostfetchExample(some_int=i) for i in range(5)]
    session.add_all(postfetch_examples)
    session.flush()
    assert {pfe.id for pfe in postfetch_examples} == {0, 1, 2, 3, 4}
```

**Comment by Jafar-Badour-Daloopa** [comment]
It seems this doesnt work
probably reverted in later versions

**Pull request merged**